.gitignore

```diff
@@ -3,3 +3,9 @@ node_modules
 
 # logs
 npm-debug.log
+
+# Docker
```

Comment:
Can you remove all of this after here? I don't see how it's necessary.

Reply:
Sure, will do. Just added it for internal testing purposes